bin/generate-specifications.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+import { api, session, file } from "@shopify/cli-kit";
+import { fileURLToPath } from 'node:url';
+import path from 'node:path';
+import fs from 'node:fs';
+
+const apiKey = "" // Add API KEY with full access
```

Comment:
the problem is still about how to automate this 🤔 
And how can we be sure to always use an updated app with full-access, should this be generated regularly by us? should it be generated at release time? probably not because it might introduce errors?